src/walletSdk/Anchor/Sep24.ts

```diff
@@ -61,6 +61,10 @@ export class Sep24 {
    * @param {ExtraFields} [params.extraFields] - Additional fields for the request.
    * @param {Memo} [params.destinationMemo] - Memo information for the destination account.
    * @param {string} [params.destinationAccount] - The destination account for the deposit.
+   * @param {string} [params.callback] - The callback URL or postMessage the anchor should POST to
```

Comment:
@Ifropc is `postMessage` something that's used / we should support?

and if so what's an example of one? just a JSON string?

Reply:
We should not support `postMessage`. This is an error-prone way of signaling to the wallet to close the webview, and there are more straightforward approaches to this that have better user experiences as well.

Reply:
1. We are deprecating postMessage
2. We are also deprecating approach of passing callbacks "as is" to the request parameter. I'm working on drafting new approach, where callbacks are passed outside of the request.  
So let's hold on this PR for now